### ISSUE TITLE
Cannot read property 'basePath' of undefined

### ISSUE THREAD

**Issue by czp3009** [HubSpot/hubspot-api-nodejs#11]
just run this code, no extra step to reproduce this problem:

main.js
```js
const hubspot = require('@hubspot/api-client')
const hubspotClient = new hubspot.Client({
    apiKey: '123456789',
    useLimiter: false
})
hubspotClient.crm.companies.getAll().then(console.log)
```
log:

```txt
~/WebstormProjects/untitled > node main.js
(node:9964) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'basePath' of undefined
    at /home/czp/WebstormProjects/untitled/node_modules/@hubspot/api-client/lib/codegen/crm/companies/api/basicApi.js:328:39
    at Generator.next (<anonymous>)
    at /home/czp/WebstormProjects/untitled/node_modules/@hubspot/api-client/lib/codegen/crm/companies/api/basicApi.js:19:71
    at new Promise (<anonymous>)
    at __awaiter (/home/czp/WebstormProjects/untitled/node_modules/@hubspot/api-client/lib/codegen/crm/companies/api/basicApi.js:15:12)
    at getPage (/home/czp/WebstormProjects/untitled/node_modules/@hubspot/api-client/lib/codegen/crm/companies/api/basicApi.js:327:16)
    at Client.<anonymous> (/home/czp/WebstormProjects/untitled/node_modules/@hubspot/api-client/lib/src/client.js:428:34)
    at Generator.next (<anonymous>)
    at /home/czp/WebstormProjects/untitled/node_modules/@hubspot/api-client/lib/src/client.js:8:71
    at new Promise (<anonymous>)
(node:9964) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:9964) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

node version(test in those version):  v10.15.1, v12.18.1
@hubspot/api-client version: ^1.0.0-beta

package.json
```json
{
  "name": "untitled",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
    "start": "node main.js"
  },
  "dependencies": {
    "@hubspot/api-client": "^1.0.0-beta"
  }
}
```

**Comment by AProts** [comment]
Hi @czp3009 ,

Thanks for catching this. The issue will be fixed in the upcoming release. Till that time, as a workaround, please do not switch off limiter for new client instances.

Kind Regards,
Andrii

**Comment by AProts** [comment]
fixed in 2.1.1
